lakefile.lean

```diff
@@ -36,6 +21,4 @@ require «doc-gen4» from git
 lean_lib FLT where
   globs := #[
     .andSubmodules `FLT
-    ]
---  moreLeanArgs := moreLeanArgs
---  weakLeanArgs := weakLeanArgs
```

Comment:
Previously these weren't actually taking effect.

Reply:
Oh that would explain why autoimplicits were on anyway :-) Thanks for fixing things up!